components/dashboard/src/App.tsx

```diff
@@ -26,20 +27,24 @@ const App: FC = () => {
     const history = useHistory();
     const location = useLocation();
     const search = useQueryParams();
+    const { isDark, setIsDark } = useTheme();
 
     useEffect(() => {
         const onKeyDown = (event: KeyboardEvent) => {
             if ((event.metaKey || event.ctrlKey) && event.key === "o") {
                 event.preventDefault();
                 history.push("/new");
                 return;
+            } else if (event.metaKey && event.ctrlKey && event.shiftKey && event.key === "M") {
```

Comment:
Added this back to just ease checking dark/light mode during development. Hopefully this combo of keys doesn't conflict with anything.